staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -36,13 +36,13 @@ import (
 	utiltrace "k8s.io/apiserver/pkg/util/trace"
 )
 
-func createHandler(r rest.NamedCreater, scope RequestScope, admit admission.Interface, includeName bool) http.HandlerFunc {
+func createHandler(r rest.NamedCreater, scope RequestScope, admit admission.Interface, includeName bool, features RESTHandlerFeatures) http.HandlerFunc {
 	return func(w http.ResponseWriter, req *http.Request) {
 		// For performance tracking purposes.
 		trace := utiltrace.New("Create " + req.URL.Path)
 		defer trace.LogIfLong(500 * time.Millisecond)
 
-		if isDryRun(req.URL) {
+		if isDryRun(req.URL) && !features.AllowDryRun {
```

Comment:
> Yeah, I've implemented this because it helps me test the feature (mostly the storage piece, not the admission chain piece). I don't want to wait for the admission part to be complete before I can test the storage? But yes, I agree it's not needed here, since I don't have these tests yet.

Presumable in a branch where you wired storage to understand it?  How about moving this one line in each of the verbs to your branch where you're doing that.  An accident for this would be quite severe.

Reply:
Yeah, that makes sense (also why it's a different commit ;-)